src/translations/en-GB/commerce.php

```diff
@@ -225,13 +217,12 @@
     'Enter weight' => 'Enter weight',
     'Exclude this discount for products that are already on sale' => 'Exclude this discount for products that are already on sale',
     'Expand' => 'Expand',
-    'Expired' => 'Expired',
-    'Expiry Date' => 'Expiry Date',
-    'Export' => 'Export',
+    'Expired' => 'Expand',
```

Comment:
@angrybrad Is this a bug in the automation or a bad translation? It's a regression.